build/Dockerfile

```diff
@@ -1,4 +1,10 @@
-FROM registry.redhat.io/ubi8/go-toolset:latest as builder
+# We cannot use the official golang images from docker.io because it's blocked
+# on our CI. Instead, we use image maintained by Quay team which is updated every
+# few hours automatically from their GitHub. For more info:
+#
+#  https://github.com/quay/claircore/actions/workflows/golang-image.yml
+#  https://github.com/quay/claircore/blob/main/.github/workflows/golang-image.yml
+FROM quay.io/projectquay/golang:1.18 as builder
```

Comment:
pulled this from https://github.com/RHEnVision/provisioning-backend/blob/main/build/Upstream.dockerfile